lib/response.js

```diff
@@ -251,7 +251,9 @@ module.exports = {
 
   redirect: function(url, alt){
     // location
+    const mountPath = this.ctx.mountPath;
```

Comment:
-1.

override `redirect` in `koa-mount` is much better than add this logic here, or write a `koa-mount-redirect` for this feature alone.